kuma/core/middleware.py

```diff
@@ -53,7 +54,7 @@ def process_request(self, request):
             return response
 
         request.path_info = '/' + prefixer.shortened_path
-        request.locale = prefixer.locale
+        request.locale = prefixer.locale or settings.LANGUAGE_CODE
```

Comment:
@jezdez I wanted to call your attention to this. It fixes URLs w/o a locale in them (those in `LANGUAGE_URL_IGNORED_PATHS`) that were highlighted by the test failures by defaulting to `en-US`. We can keep or remove this as part of the `request.locale` -> `request.LANGUAGE_CODE` changes.